packages/common/containerd.py

```diff
@@ -0,0 +1,73 @@
+#! /bin/env python
```

Comment:
Please specify either `python2` or `python3` ditto for the other file.

Also, about the location, I would put the files under `packages/common/metalk8s-sosreport` (or even `packages/common/metalk8s-sosreport/src`).

In case we have more packages in the future we shouldn't put stuff directly under `common`.